rest-api/api/biobank_order_api.py

```diff
@@ -15,3 +15,11 @@ def post(self, p_id):
   @auth_required(PTC_AND_HEALTHPRO)
   def get(self, p_id, bo_id=None):  # pylint: disable=unused-argument
     return super(BiobankOrderApi, self).get(bo_id)
+
+  @auth_required(HEALTHPRO)
```

Comment:
So we're supporting PUT and PATCH? Which one will get used under what circumstances?